.claude/rules/code-review.md

```
@@ -0,0 +1,8 @@
+### 코드 리뷰 (Pn 룰)
+| 우선순위 | 설명 |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)